test/crud-with-options.test.js

```diff
@@ -520,6 +521,27 @@ describe('crud-with-options', function() {
       });
     });
   });
+
+  describe('updateAttributes', function() {
+    beforeEach(seed);
+    it('shouldn\'t null not given embedded document properties', function() {
+      if (db.name !== 'mongodb') {
+        console.log('No need to check except for Mongo');
+      }
+      return User.findOne({where: {name: 'John Lennon'}})
+        .then(function(user) {
+          return user.updateAttributes({address: {city: 'Volos'}});
+        })
+        .then(function() {
+          return User.findOne({where: {name: 'John Lennon'}}); // retrieve the user again from the db
+        })
+        .then(function(updatedUser) {
+          updatedUser.address.city.should.equal('Volos');
+          should(updatedUser.address.area).not.be.exactly(null);
+          should.not.exist(updatedUser.address.area);
```

Comment:
I am rather confused about this check. What is the expected `area` value? Can you assert on what the value should be?

Reply:
Since the address.area was not initialized it should not have the value `null` which it does without the fix. So I am checking that it strictly is not null, and below that it does not exist *(it is undefined)*. I will explicitly change the check for `undefined`